lab2.js

```diff
@@ -118,33 +166,56 @@ assert((new Human()).sayHello(new Klingon()) === 'nuqneH',
 // will test your code)
 //*********************************************************
 
+var cities = ['Chicago', 'Paris', 'Los Angeles', 'Vancouver'];
+var numbers1 = [1, 2, 3, 4, 5];
+var numbers2 = [10, 15, 20, 25, 30];
+var numbers3 = [37, 1, 12, 70, 18];
+var allNumbers = [numbers1, numbers2, numbers3];
+
 function lastLetterSort(stringArray) {
+
   function byLastLetter(a, b) {
-    //TODO: implement me. sort the strings in alphabetical
-    // order using their last letter
-    // Read this about how the sort function works:
-    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
-    // this byLastLetter function is a "compare function"
-    // And check out the "comparing strings" section  here:
-    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
+
+    if ((a.charAt(a.length - 1)) < (b.charAt(b.length - 1))) {
+      return -1;
```

Comment:
don't need these - you can just return the result of subtracting one last letter from the other. That result will either be negative, positive, or 0 - and that's what .sort is looking for.